### PR TITLE
Add hook that checks if Template supports QuickCreate

### PR DESCRIPTION
**Pull request by glekner**
## 📝 Description

Added `useIsTemplateSupportsQuickCreate` that detects if QuickCreate is supported by a given template.
The Drawer will show a skeleton until the hook detection completes

## 🎥 Demo
Here is a Demo with Network Throttled to demonstrate the loading experience

https://user-images.githubusercontent.com/24938324/154291800-e38f7d6c-22c1-4db2-8b20-60464f853b98.mov

### CODE REVIEW

File: src/views/catalog/templatescatalog/hooks/useIsTemplateSupportsQuickCreate.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+
+import { V1Template } from '@kubevirt-ui/kubevirt-api/console';
+import { k8sCreate } from '@openshift-console/dynamic-plugin-sdk';
+import { K8sModel } from '@openshift-console/dynamic-plugin-sdk/lib/api/common-types';
+
+export const ProcessedTemplatesModel: K8sModel = {
```
Comment (glekner) [inline src/views/catalog/templatescatalog/hooks/useIsTemplateSupportsQuickCreate.ts:7]: @yaacov do we need this model?

Comment (yaacov) [reply src/views/catalog/templatescatalog/hooks/useIsTemplateSupportsQuickCreate.ts:7]: do the sdk expose any models ?

Comment (glekner) [reply src/views/catalog/templatescatalog/hooks/useIsTemplateSupportsQuickCreate.ts:7]: the sdk doesn't expose models

Comment (yaacov) [reply src/views/catalog/templatescatalog/hooks/useIsTemplateSupportsQuickCreate.ts:7]: than we will add it to the API repo :+1:

File: webpack.config.ts [hunk 2]
```diff
@@ -57,7 +57,7 @@ const config: Configuration = {
     rules: [
       {
         test: /\.(jsx?|tsx?)$/,
-        exclude: /node_modules/,
+        exclude: /node_modules\/(?!(@kubevirt-ui)\/kubevirt-api).*/,
```
Comment (yaacov) [inline webpack.config.ts:60]: shouldn't this be in a separate PR ?

File: src/views/catalog/templatescatalog/hooks/useIsTemplateSupportsQuickCreate.ts [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  labelPlural: 'Processed Templates',
+  apiVersion: 'v1',
+  apiGroup: 'template.openshift.io',
+  plural: 'processedtemplates',
+  abbr: 'PT',
+  namespaced: true,
+  kind: 'Template',
+};
+
+export const useIsTemplateSupportsQuickCreate = (template: V1Template): [boolean, boolean, any] => {
```
Comment (yaacov) [inline src/views/catalog/templatescatalog/hooks/useIsTemplateSupportsQuickCreate.ts:18]: It will be nice to have this hook return the VM to be crated instead of just if an error happened
the name can then be more informative, it will tell what we actullay doing ( useProcessedTemplate ? )
and the user will use the return error as indicator if the template is ready to be created as is, WDYT ?

**Comment by glekner** [comment]
please lgtm @yaacov
